src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -222,13 +222,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
         Private Function ScanNextCharAsToken(leadingTrivia As SyntaxList(Of VisualBasicSyntaxNode)) As SyntaxToken
             Dim token As SyntaxToken
-
-            If Not CanGet() Then
+            Dim c As Char
+            If Not Peep(c) Then
```

Comment:
I guess, we can assume that CanGet generally returns true and as such folding CanGet and following Peek would be  ok. 
In a case of "If Not CanGet()", CanGet seems would be substantially cheaper than "If Not Peep()"